gpkit/signomial_program.py

```diff
@@ -116,7 +116,9 @@ def localsolve(self, solver=None, verbosity=1, x0=None, reltol=1e-4,
 
         while (iterations < iteration_limit
                and (not (cost and prevcost)
-                    or abs(prevcost-cost)/(prevcost + cost) > reltol)):
+                    or abs(prevcost-cost)/(prevcost + cost) > reltol)
+               and (cost > 1e-10 or cost == None)):
```

Comment:
`None` is funky. and might always return False to ==; the preferred check is `cost is None`

Alternately, 

```and (not (cost and prevcost)
            or abs(prevcost-cost)/(prevcost + cost) > reltol
            and cost > 1e-10)
```
This doesn't need the None check because it comes after an `and`; if the first bit is False, Python won't even try the second bit.

Unrelatedly, to work on my Windows machine this needs an eps of `1e-6`